globals.go

```diff
@@ -509,19 +510,34 @@ type PermissionLevel struct {
 }
 
 var (
+	// SuperUserPermissions resource ID.
+	SuperUserPermissionsID = "super_user"
+
+	// Admin permissions.
+	PermissionAdminSettings = "admin_settings"
 	// Project permissions.
+	PermissionProjectCreate    = "project_create"
```

Comment:
The admin permissions should be grouped together to make it clear they aren't related to the project/distro permissions